packages/twenty-front/src/modules/activities/components/ParticipantChip.tsx

```diff
@@ -0,0 +1,70 @@
+import styled from '@emotion/styled';
+
+import { getDisplayNameFromParticipant } from '@/activities/emails/utils/getDisplayNameFromParticipant';
+import { CoreObjectNameSingular } from '@/object-metadata/types/CoreObjectNameSingular';
+import { RecordChip } from '@/object-record/components/RecordChip';
+import { Avatar } from '@/users/components/Avatar';
+
+const StyledAvatar = styled(Avatar)`
+  margin: ${({ theme }) => theme.spacing(0, 1)};
+`;
+
+const StyledSenderName = styled.span<{ variant?: 'default' | 'bold' }>`
+  color: ${({ theme }) => theme.font.color.primary};
+  font-size: ${({ theme }) => theme.font.size.sm};
+  font-weight: ${({ theme, variant }) =>
+    variant === 'bold' ? theme.font.weight.medium : theme.font.weight.regular};
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const StyledContainer = styled.div`
+  align-items: flex-start;
+  display: flex;
+`;
+
+const StyledRecordChip = styled(RecordChip)<{ variant: 'default' | 'bold' }>`
+  font-weight: ${({ theme, variant }) =>
+    variant === 'bold' ? theme.font.weight.medium : theme.font.weight.regular};
+`;
+
+export const ParticipantChip = ({
```

Comment:
can this be abstracted (not necessarily in this PR) to a regular chip? Feels weird to have something specific for Participant